scripts/set-assignees.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -euo pipefail
+LANGUAGE=$1
+
+# extracts de from de-german
+LANG_CODE=${LANGUAGE:0:2}
+number_of_maintainers=`jq '.maintainers | length' < ./.github-workflow/langs/$LANG_CODE.json`
+
+assignee=$((($RANDOM%$number_of_maintainers)))
+reviewer=$((($RANDOM%$number_of_maintainers)))
+
+while [ "$assignee" -eq "$reviewer" ]
+do
+    reviewer=$((($RANDOM%$number_of_maintainers)))
+done
```

Comment:
This will be an infinite loop if there's only one maintainer :)

We currently have a rule that there should be at least two of them, but still, there's nothing technically preventing anyone from putting just one in the JSON file (who knows, maybe we'll need to do an exception some day). The script could handle that by leaving reviewer unassigned if there's only one person.

Reply:
done